src/olympia/ratings/serializers.py

```diff
@@ -7,19 +7,29 @@
 
 import six
 
+from bleach.linkifier import TLDS
 from rest_framework import serializers
 from rest_framework.relations import PrimaryKeyRelatedField
 
 from olympia.accounts.serializers import BaseUserSerializer
 from olympia.addons.serializers import (
     SimpleAddonSerializer, SimpleVersionSerializer)
 from olympia.api.utils import is_gate_active
-from olympia.ratings.forms import RatingForm
-from olympia.ratings.models import Rating
+from olympia.ratings.models import Rating, RatingFlag
 from olympia.ratings.utils import maybe_check_with_akismet
 from olympia.versions.models import Version
 
 
+# This matches the following three types of patterns:
+# http://... or https://..., generic domain names, and IPv4 octets. It does not
+# match IPv6 addresses or long strings such as "example dot com".
+link_pattern = re.compile(
+    r'((://)|'  # Protocols (e.g.: http://)
+    r'((\d{1,3}\.){3}(\d{1,3}))|'
+    r'([0-9a-z\-%%]+\.(%s)))' % '|'.join(TLDS),
+    (re.I | re.U | re.M))
```

Comment:
I get the feeling that you may be able to re-use `django.core.validators:URLValidator.regex` instead which allows much more stuff and is also much better tested (e.g, way more thorough tld restrictions etc).

https://github.com/django/django/blob/master/django/core/validators.py#L95

Reply:
I'm perfectly aware that this only got copied so treat this as a nitpick and nothing blocking merge.

Reply:
I just tried it but `URLValidator`'s regex has `^` so it doesn't match.